packages/dart/sshnoports/lib/src/noports_cli/activate/np_activate_params.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:sshnoports/src/noports_cli/util/constants.dart';
+import 'package:sshnoports/src/noports_cli/util/regex.dart';
+
+class NoportsParams {
```

Comment:
Shouldn't be called NoportsParams; the `noports` binary is ultimately intended as a wrapper which starts up any of the noports programs. It has no parameters of its own. Since this class is specifically about the `noports activate` command, it should be named accordingly.